src/main/java/TaskManager.java

```diff
@@ -0,0 +1,47 @@
+public class TaskManager {
+    private static int taskNo = 0;
+    final private static int maxTasks = 100;
+    final static Task[] tasks = new Task[maxTasks];
+
+    static void addTask(Task task) {
+        if (taskNo < maxTasks) {
+            tasks[taskNo] = task;
+            taskNo++;
+
+            System.out.print("Got it. I've added this task:\n");
+            System.out.printf("   %s\n", task);
+            System.out.printf("Now you have %d tasks in the list.\n", taskNo);
+        }
+    }
+
+    static void addTodo(String todoName) {
+        Todo todo = new Todo(todoName);
+        addTask(todo);
+    }
+
+    static void addDeadline(String deadlineName, String deadlineDue) {
```

Comment:
Perhaps for the second parameter, you can use the same name as in Deadline.java (in this case it is dueDate), or a similar one in style with deadlineName